src/styles/index.scss

```diff
@@ -1,3 +1,212 @@
+  width: 726px;
+  padding: 10px;
+  display: flex;
+
+  &__day {
+    background-color: #eee;
+    border: 1px solid #000;
+    justify-content: center;
+    align-items: center;
+    width: 100px;
+    height: 100px;
+    display: flex;
+
+    &:first-child::before {
+      content: '1';
+    }
+
+    &:nth-child(2)::before {
+      content: '2';
+    }
+
+    &:nth-child(3)::before {
+      content: '3';
+    }
+
+    &:nth-child(4)::before {
+      content: '4';
+    }
+
+    &:nth-child(5)::before {
+      content: '5';
+    }
+
+    &:nth-child(6)::before {
+      content: '6';
+    }
+
+    &:nth-child(7)::before {
+      content: '7';
+    }
+
+    &:nth-child(8)::before {
+      content: '8';
+    }
+
+    &:nth-child(9)::before {
+      content: '9';
+    }
+
+    &:nth-child(10)::before {
+      content: '10';
+    }
+
+    &:nth-child(11)::before {
+      content: '11';
+    }
+
+    &:nth-child(12)::before {
+      content: '12';
+    }
+
+    &:nth-child(13)::before {
+      content: '13';
+    }
+
+    &:nth-child(14)::before {
+      content: '14';
+    }
+
+    &:nth-child(15)::before {
+      content: '15';
+    }
+
+    &:nth-child(16)::before {
+      content: '16';
+    }
+
+    &:nth-child(17)::before {
+      content: '17';
+    }
+
+    &:nth-child(18)::before {
+      content: '18';
+    }
+
+    &:nth-child(19)::before {
+      content: '19';
+    }
+
+    &:nth-child(20)::before {
+      content: '20';
+    }
+
+    &:nth-child(21)::before {
+      content: '21';
+    }
+
+    &:nth-child(22)::before {
+      content: '22';
+    }
+
+    &:nth-child(23)::before {
+      content: '23';
+    }
+
+    &:nth-child(24)::before {
+      content: '24';
+    }
+
+    &:nth-child(25)::before {
+      content: '25';
+    }
+
+    &:nth-child(26)::before {
+      content: '26';
+    }
+
+    &:nth-child(27)::before {
+      content: '27';
+    }
+
+    &:nth-child(28)::before {
+      content: '28';
+    }
+
+    &:nth-child(29)::before {
+      content: '29';
+    }
+
+    &:nth-child(30)::before {
+      content: '30';
+    }
+
+    &:nth-child(31)::before {
+      content: '31';
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transition-duration: 0.5s;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+$days: (mon, tue, wed, thu, fri, sat, sun);
+
+// .calendar--start-day-mon :first-child{
+//   margin-left: 100 * 0 + 1 * 0;
+// }
+
+// .calendar--start-day-tue :first-child{
+//   margin-left: 100 * 1 + 1 * 1;
+// }
+
+// .calendar--start-day-wed :first-child{
+//   margin-left: 100 * 2 + 1 * 2;
+// }
+
+// .calendar--start-day-thu :first-child{
+//   margin-left: 100 * 3 + 1 * 3;
+// }
+
+// .calendar--start-day-fri :first-child{
+//   margin-left: 100 * 4 + 1 * 4;
+// }
+
+// .calendar--start-day-sat :first-child{
+//   margin-left: 100 * 5 + 1 * 5;
+// }
+
+// .calendar--start-day-sun :first-child{
+//   margin-left: 100 * 6 + 1 * 6;
+// }
+
+@for $i from 0 through 6 {
```

Comment:
Move indexes to this variable and use it here with `@each` loop

Reply:
<img width="413" alt="image" src="https://github.com/user-attachments/assets/b5bb0666-59c1-4177-bc41-3c2d6e39dbda">

Reply:
idk how to use @each here, i tried. can you help?